### DISCUSSION TITLE
openwrt-23.05分支刚刚编译出来，客户端收不到ipv6地址

### DISCUSSION THREAD

**Discussion by Pacio** [immortalwrt/immortalwrt#1370] [Q&A]
### Describe the bug

云编译的，就勾选了三个插件, dockerman / homeproxy/upnp 其他设置全默认

编译出来的固件全默认没动，只填入宽带账户密码，拨号显示已经有了ipv6地址和pd，但是客户端比如电脑，检测不到ipv6地址

### ImmortalWrt version

最新源码

### ImmortalWrt release

23.05

### ImmortalWrt target/subtarget

X86/64

### Device

Intel(R) Pentium(R) Gold 8505

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
去掉 ula 即可。

**Reply by Pacio** [reply]
> 去掉 ula 即可。
去掉了啊

**Reply by Pacio** [reply]
> 去掉 ula 即可。

全都一样的设置，换成[ImmortalWrt Firmware Selector](https://firmware-selector.immortalwrt.org/)的固件就正常了

**Comment by 1715173329** [comment]
docker 造成的，参阅 https://github.com/openwrt/packages/commit/55e689176ec52afde0cda0ee2b48dbff349d6c6c 。

**Reply by 363137946** [reply]
添加了docker编译出来，ipv6-test不能通过，要怎么解决

**Comment by wyndam** [comment]
您的upnp有用吗？我装的upnp没有任何反应啊，没看到正在连接的

**Reply by Pacio** [reply]
> 您的upnp有用吗？我装的upnp没有任何反应啊，没看到正在连接的

没用，只有lede和istoreos我这边试了可以，而且流量监控也用不了
